spec/models/bookmark_query_spec.rb

```diff
@@ -36,24 +51,28 @@
 
   it "should not return bookmarks of hidden objects" do
     q = BookmarkQuery.new
-    expect(q.exclusion_filters).to include(has_parent: { parent_type: 'bookmarkable', query: { term: { hidden_by_admin: 'true' } } })
+    parent_filter = q.exclusion_filters.first { |f| f.key? :has_parent }
```

Comment:
It looks as if there's some repeated code that could possibly be extracted into a `before :each` - unless that creates more problems than it's worth